setup.py

```diff
@@ -22,7 +22,7 @@
                       'opencivicdata>=3.1.0',
                       'pytz>=2015.4',
                       'django-haystack>=2.8.0,<2.9',
-                      'Django>=2.0,<2.2',
+                      'Django>=2.1,<2.2',
```

Comment:
`json_script` was introduced in Django 2.1. How badly do we want to support Django 2.0.x? If we do, we can always copy the logic for `json_script` from the Django source and use that if the installed Django version is less than 2.1.